core/blockchain_test.go

```diff
@@ -423,8 +425,11 @@ func TestUngracefulAsyncShutdown(t *testing.T) {
 	// Ensure that key1 has some funds in the genesis block.
 	genesisBalance := big.NewInt(1000000)
 	gspec := &Genesis{
-		Config: &params.ChainConfig{HomesteadBlock: new(big.Int)},
-		Alloc:  types.GenesisAlloc{addr1: {Balance: genesisBalance}},
+		Config: params.WithExtra(
+			&params.ChainConfig{HomesteadBlock: new(big.Int)},
+			&extras.ChainConfig{},
+		),
```

Comment:
I think this can be reverted to

```suggestion
		Config: &params.ChainConfig{HomesteadBlock: new(big.Int)},
```

Reply:
Bundled in #862